UnityProject/Assets/Scripts/Items/Stackable.cs

```diff
@@ -209,13 +215,14 @@ public int ServerIncrease(int increase)
 	[Server]
 	public GameObject ServerRemoveOne()
 	{
-		SyncAmount(amount, amount - 1);
-		if (amount <= 0)
+		if ((amount-1) <= 0)
 		{
 			return gameObject;
 		}
+		SyncAmount(amount, amount - 1);
 
 		var spawnInfo = Spawn.ServerPrefab(prefab, gameObject.transform.position, gameObject.transform);
+		spawnInfo.GameObject.GetComponent<Stackable>().ServerSetAmount(1);
 		return spawnInfo.GameObject;
 	}
```

Comment:
Was this broken for the items which were using this before?

Reply:
yeah, it could return something with 0 stacked, then when it tried recombining it disappear

Reply:
ah cool, bug fixed woo (Dont think theres an issue open for it)